### ISSUE TITLE
helptags list doesn’t display helptags for packages added after launch

### ISSUE THREAD

**Issue by TSFoster** [neoclide/coc-lists#141]
Hi,

I have opt packages that I only `packadd` to vim when loading specific filetypes (e.g. I have the line `vim.cmd "packadd emmet-vim"` in `ftplugin/html.lua`). While I can do `:help emmet` after loading an HTML file into a buffer, I can't find the same doc with `:CocList helptags`.

I'm adding this to this repo because the problem I'm experiencing is related to the helptags list. Based on a couple of quick `grep`s, I think the problem might be that coc.nvim's `workspace.env` doesn't update, so `workspace.env.runtimepath` doesn't reflect changes to `&rtp`. Do I have that right?

**Comment by chemzqm** [comment]
No, it's automatically updated by use `OptionSet` autocmd, the autocmd may not fired or you don't have tags file created in doc folder.
